elements/pfe-label/pfe-label.scss

```diff
@@ -0,0 +1,142 @@
+@use "@patternfly/pfe-sass" as *;
+@use "@patternfly/patternfly/sass-utilities/colors" as *;
+
+@include configure(
```

Comment:
I can nuke the sass completely if we feel like that is how we want to move forward.

Reply:
I'd recommend keeping the .scss filetype and the namespace imports, just in case and to simplify tooling

other than that why not keep it all CSS? modern css is amazing, plus it would be way more readable